25-minimize.ks

```diff
@@ -4,9 +4,14 @@
 # Some ideas from:
 #
 # https://github.com/weldr/lorax/blob/rhel7-branch/share/runtime-cleanup.tmpl
+# https://github.com/weldr/lorax/blob/rhel9-branch/share/templates.d/99-generic/runtime-cleanup.tmpl
 #
 %post
 
+#
+echo " * Remove /usr/share/i18n"
+rm -rf /usr/share/i18n
```

Comment:
On my Fedora and a clean CentOS 9 container these are empty. If they're big, then I wonder what makes them big. You can use `rpm -qf /path/to/file` to find out which RPM owns that file.